### PR TITLE
Add .venv* to .gitignore

### PR DESCRIPTION
**Pull request by ShadowJonathan**
> This is so files automatically generated by `virtualenv` and pycharm will be ignored

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)
* [x] Code style is correct (run the [linters](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#code-style))

`Signed-off-by: Jonathan de Jong <jonathan@automatia.nl>`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,5 @@ _trial_temp*/
 /docs/build/
 /htmlcov
 /pip-wheel-metadata/
+
+.venv*
```
Comment (anoadragon453) [inline .gitignore:46]: Could you move this next to `/env/` above?

Comment (anoadragon453) [inline .gitignore:46]: It'd be nice to also match the `/xxx/` form above, unless you meant to match on multiple possible directory names.

**Review by anoadragon453 — CHANGES_REQUESTED** [review]
lgtm besides some small nits. I assume this will save some headaches down the line for some people :)

File: .gitignore (hunk 1 above)
Comment (ShadowJonathan) [reply .gitignore:46]: i think `/.venv*/` would still work, then, lemme try

**Review by anoadragon453 — APPROVED** [review]

**Pull request merged**